website/content/userguide/permanent-data.markdown

```diff
@@ -0,0 +1,97 @@
+---
+title: "Access permanent data"
+menuTitle: "Access permanent data"
+weight: 49
+---
+
+## Create the filesystem
```

Comment:
This section currently discusses creating the root file system.

In general, instead of working with the device nodes directly (e.g. `/dev/sda`), it’s better to use the descriptive names within `/dev/disk`, e.g. `sudo mkfs.ext4 /dev/disk/by-partuuid/2e18c40c-04`, as we do elsewhere in the documentation.

That said, it’s even easier to just include the new `github.com/gokrazy/mkfs` Go package and have it create the `/perm` partition on the device itself :)

Could you update the section to use the new way? You can copy from the https://github.com/gokrazy/mkfs README file